src/tracing/instrumentation/mysql.js

```diff
@@ -23,7 +23,9 @@ function instrumentMysql(mysql) {
 
 function instrumentMysql2(mysql) {
   instrumentConnection(mysql.Connection.prototype);
-  instrumentPool(mysql.Pool.prototype);
+  if ( typeof mysql.Pool !== 'undefined' && mysql.Pool ) {
```

Comment:
You can shorten this to 

```javascript
if (mysql.Pool) {
  // …
}
```

as this will cover all truthy cases. You could even shorten it to:

```javascript
mysql.Pool && instrumentPool(mysql.Pool.prototype);
```